pkg/kinflate/tree/builder.go

```diff
@@ -144,21 +156,21 @@ func loadKObjectFromPath(path string, into types.KObject) error {
 			return nil
 		}
 
-		err = loadKObjectFromFile(filepath, into)
+		err = l.LoadKObjectFromFile(filepath, into)
 		return nil
 	})
 	return e
 }
 
-func loadKObjectFromFile(filename string, into types.KObject) error {
-	f, err := os.Stat(filename)
+func (l *Loader) LoadKObjectFromFile(filename string, into types.KObject) error {
```

Comment:
is this exporting a lot more symbols? It's hard to know, by looking at this change, what is the impact on the godoc though

Reply:
There is no need to make this method public. I will change it.